scalr/resource_scalr_webhook.go

```diff
@@ -433,7 +433,6 @@ func readNewWebhook(ctx context.Context, d *schema.ResourceData, scalrClient *sc
 	_ = d.Set("enabled", webhook.Enabled)
 	_ = d.Set("last_triggered_at", webhook.LastTriggeredAt)
 	_ = d.Set("url", webhook.Url)
-	_ = d.Set("secret_key", webhook.SecretKey)
```

Comment:
What about the `scalr_webhook` datasource?

Reply:
@petroprotsakh Updated the AC in the ticket. This attribute will be removed from the data source.